spec/integration_spec.rb

```diff
@@ -116,13 +120,14 @@
                             '--root-url https://realm.io/docs/swift/' \
                             "#{realm_version}/api/ " \
                             '--xcodebuild-arguments ' \
-                            '-project,RealmSwift.xcodeproj,-dry-run'
+                            '-project,RealmSwift.xcodeproj,-dry-run ' \
+                            '--template-directory "docs/templates/swift" '
     end
 
-    describe 'Creates docs for a podspec with dependencies and subspecs' do
-      behaves_like cli_spec 'document_moya_podspec',
-                            '--podspec=Moya.podspec'
-    end
+    # describe 'Creates docs for a podspec with dependencies and subspecs' do
```

Comment:
My assumption is that there's a lot more broken to support Swift 1.2 within scope of this PR from a SourceKit/SourceKitten point of view, but I haven't actually explored that fully, so I'll do that now.